app/controllers/application_controller.rb

```diff
@@ -1,5 +1,31 @@
 class ApplicationController < ActionController::Base
+  ALLOWED_ORIGINS = ENV.fetch('ALLOWED_ORIGINS').split(',').freeze
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   # protect_from_forgery with: :exception
+
+  after_action :cors_set_access_control_headers
```

Comment:
because browser requests will refuse to use the response if the correct access isn't granted in the headers of a cors request. But if we end up moving the authentication to Rainforest, then this `after_action` will only need to happen on the one endpoint that the frontend uses.